### PR TITLE
11.0.0 base branch (PR = flatcap-july into develop)

### PR DESCRIPTION
**Pull request by Stanton**
DO NOT MERGE UNTIL BETA PHASE ENDS.

### CODE REVIEW

File: stylesheets/_component.notifications.scss [hunk 1]
```diff
@@ -1,7 +1,7 @@
 @import 'component.toolbar';
 
 .notifications {
-    float: right;
+    // float: right;
```
Comment (jamesjacobs) [inline stylesheets/_component.notifications.scss:4]: Remove?

File: stylesheets/_component.navigation.scss [hunk 2]
```diff
@@ -179,9 +179,11 @@
     border-radius: 4px;
     color: $nav-color-dark;
     display: inline-block;
+    height: 32px;
```
Comment (jamesjacobs) [inline stylesheets/_component.navigation.scss:182]: Why do we need to set a height?

File: js/TabEnhancements/TabEnhancements.js [hunk 3]
```diff
@@ -18,47 +18,22 @@ class TabEnhancements {
         // Make sure tab panes are at least as high as the tab list (cms legacy tabs list)
         this.$html.find('.tabs > .tabs__content > .tab__pane').css('min-height', this.$html.find('.tabs__list').height());
 
-        // Switch the first active tab to use <main> with the skip target
-        this.changeActiveTabElementToMain(this.$html.find('.tab__pane.is-active'));
+        // Remove skip target IDs set in markup
```
Comment (jamesjacobs) [inline js/TabEnhancements/TabEnhancements.js:21]: These changes are already in develop, so I'm confused why these are showing as a diff?

File: stylesheets/_component.rules.scss [hunk 4]
```diff
@@ -498,7 +519,7 @@ td.table__form .rules + .form__group:last-of-type {
 
     .rule-block--if,
     .rule-block--else-if {
-        background-color: lighten($rule-color-if, 40);
+        background-color: lighten($rule-color-if, 41);
```
Comment (jamesjacobs) [inline stylesheets/_component.rules.scss:522]: These bg color changes are also in develop so why are these showing in the diff?

File: stylesheets/_layout.settings.scss [hunk 5]
```diff
@@ -5,7 +5,7 @@
 .tab__settings {
     @include clear-fix(true);
 
-    background-color: color(background);
+    background-color: color(background, light);
```
Comment (jamesjacobs) [inline stylesheets/_layout.settings.scss:8]: Another one which is already in develop

File: stylesheets/_palette.base.scss [hunk 6]
```diff
@@ -186,7 +186,7 @@ $palette-alias: (
     background: (
         base:      map-get(map-get($palette-monochromes, gray), off-white),
         dark:      map-get(map-get($palette-monochromes, gray), lightest),
-        light:     lighten(map-get(map-get($palette-monochromes, gray), off-white), 2),
+        light:     lighten(map-get(map-get($palette-monochromes, gray), off-white), 2.1),
```
Comment (jamesjacobs) [inline stylesheets/_palette.base.scss:189]: Already in develop

File: tests/js/web/TabEnhancements/TabEnhancementsTest.js [hunk 7]
```diff
@@ -38,7 +38,7 @@ describe('TabEnhancements', () => {
                 <div class="tab__pane is-active" id="tab1">
                     <div class="tab__container">
                         <div class="tab__inner">
-                            <div class="tab__content">
+                            <div role="main" class="tab__content id="skip-target">
```
Comment (jamesjacobs) [inline tests/js/web/TabEnhancements/TabEnhancementsTest.js:41]: Already in develop

File: views/lexicon/patterns/piano/index.html.twig [hunk 8]
```diff
@@ -10,7 +10,11 @@
 {% set heading = 'Piano' %}
 
 {% set tab_one %}
+<<<<<<< HEAD:views/lexicon/patterns/piano/deployer.html.twig
```
Comment (jamesjacobs) [inline views/lexicon/patterns/piano/index.html.twig:13]: Bad merge

File: views/lexicon/tabs/buttons_labels.html.twig [hunk 9]
```diff
@@ -13,36 +13,32 @@
...
-                'class': 'is-disabled'
-            }),
-            html.link({
-                label: html.icon('lock') ~ ' Lock',
-                disabled: true
-            })
-        ]
-    })
-}}
+    <h2 class="heading">Links & Lists</h2>
```
Comment (jamesjacobs) [inline views/lexicon/tabs/buttons_labels.html.twig:16]: &amp;

File: views/playground/workflow-designer/index.html.twig [hunk 10]
```diff
@@ -0,0 +1,57 @@
+{% extends '@pulsar/lexicon/base.html.twig' %}
+
+{% import '@pulsar/pulsar/v2/helpers/html.html.twig' as html %}
+{% import '@pulsar/pulsar/v2/helpers/form.html.twig' as form %}
+{% import '@pulsar/pulsar/v2/helpers/tabs.html.twig' as tabs %}
+{% import '@pulsar/pulsar/v2/helpers/flash.html.twig' as flash %}
+
+{% block title %}{{ parent() }}{% endblock %}
+
+<<<<<<< HEAD:views/playground/workflow-designer/index.html.twig
```
Comment (jamesjacobs) [inline views/playground/workflow-designer/index.html.twig:10]: Bad merge

File: views/pulsar/components/tab.html.twig [hunk 11]
```diff
@@ -64,7 +64,7 @@
             {{ _settings|raw }}
         </section>
         {% endif %}
-        <div class="tab__content">
+        <div class="tab__content" id="skip-target" role="main">
```
Comment (jamesjacobs) [inline views/pulsar/components/tab.html.twig:67]: Already in develop

**Review by jamesjacobs — CHANGES_REQUESTED** [review]
To discuss

File: views/playground/workflow-designer/index.html.twig (hunk 10 above)
Comment (jackbentley) [reply views/playground/workflow-designer/index.html.twig:10]: Highly recommend not resolving merge conflicts by hand and rather using something like PhpStorms conflict tool. A code review should be your last line of defence, not your first

Comment (Stanton) [reply views/playground/workflow-designer/index.html.twig:10]: It's a PR having other PRs cherry picked into it, merge conflicts are inevitable. Chill dude.

**Pull request closed without merging**